_observability-plugin/trace/trace-analytics-jaeger.md

```diff
@@ -0,0 +1,85 @@
+
+When you perform trace analytics, you can select from two data sources:
+
+- **Data Prepper** – Data ingested into OpenSearch through Data Prepper. This data source requires OpenTelemetry (OTEL) index type.
+- **Jaeger** – Trace data stored within OpenSearch as its backend. This data source requires the Jaeger index type.
+
+If you currently store your Jaeger trace data in OpenSearch, you can now use the trace analytics capability. When you ingest Jaeger data into OpenSearch, it gets stored in a different index than the OTA-generated index that gets created when you run data through the Data Prepper. You can indicate which data source on which you want to perform trace analytics with the data source selector in the Dashboards.
+
+Jaeger trace data that you can analyze includes span data, as well as service and operation endpoint data. Jaeger span data analysis requires some configuration.
+
+Each time you ingest data for Jaeger, it creates a separate index for that day. The Dashboards will show the current index that has a mapping.
+
+To learn more about Jaeger data tracing, see the [Jaeger](https://www.jaegertracing.io/) open source documentation.
+
+## Requirements
+
+Jaeger data that is ingested for OpenSearch needs to have the `--es.tags-as-fields.all=true` flag set for errors. If data is not ingested in this format it will not work for trace analytics with OpenSearch.
+
+### About Data ingestion with Jaeger indexes
+
+Trace analytics for non-Jaeger data use OTEL indexes with the naming conventions `otel-v1-apm-span-*` or `otel-v1-apm-service-map*`.
+
+Jaeger indexes follow the naming conventions `jaeger-span-*` or `jaeger-service-*`.
+
+Jaeger and OTEL indexes have different field names. Therefore when you run trace analytics, you'll need to create different queries and components depending on which index type you are using.
+{:.note}
+
+## Use trace analytics in OpenSearch Dashboards
+
+To analyze your Jaeger trace data in the Dashboards, you need to set up Trace Analytics first. To get started, see [Get started with Trace Analytics]({{site.url}}{{site.baseurl}}/observability-plugin/trace/get-started/).
+
+### Data sources
+
+You can specify either Data Prepper or Jaeger as your data source when you perform trace analytics.
+From the OpenSearch Dashboards, go to **Observability > Trace Analytics** and select Jaeger.
+
+![Select data source]({{site.url}}{{site.baseurl}}/images/trace-analytics/select-data.png)
+
+## Dashboards views
+
+After you select Jaeger for the data source, you can view all of your indexed data in the Dashboards, including Error rate, Throughput, and Services.
+### Error rate
+
+You can view the Trace error rate over time in the Dashboard, and also view the top five combinations of services and operations that have a non-zero error rate.
```

Comment:
Top 5 combinations of service and operations with the highest non-zero error **count** (we show rate separately, but it is the top 5 of count)